package.json

```diff
@@ -5,12 +5,20 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "webpack"
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix --ext .js,.jsx ."
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.30.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.23.4",
     "webpack": "^5.42.1",
     "webpack-cli": "^4.7.2"
```

Comment:
- [ ] It will be great to add all the webpack related dependencies. This will ensure webpack is well configured and the application uses all its power. It seems you forgot to add `webpack-dev-server`. ⬇️ 
![devdep_requirement](https://user-images.githubusercontent.com/32361963/126021481-0ddc72a2-33bd-49cf-a520-fa15df1f116e.png)